openedx/core/djangoapps/content/learning_sequences/views.py

```diff
@@ -201,7 +201,7 @@ def _determine_user(self, request, course_key: CourseKey) -> types.User:
             # Just like in masquerading, set real_user so that the
             # SafeSessions middleware can see that the user didn't
             # change unexpectedly.
-            target_user.real_user = request.user
+            target_user.real_user = request.user  # type: ignore
```

Comment:
Until python typing supports [intersections](https://github.com/python/typing/issues/213) there is no way to tell the type system that we want to add a `real_user` attribute onto the `User` class, so I just ignored this line.